### PR TITLE
Add method to access via Windows

### PR DESCRIPTION
**Pull request by elisehaldane**
This addition is based on feedback from the DSbD programme's TAP participants, but may also help to satisfy #18.

### CODE REVIEW

**Comment by brooksdavis** [comment]
Should we provide a link to FTDI drivers from a trustworthy source given the prevalence of malware in driver searches?

**Comment by elisehaldane** [comment]
FTDI drivers are listed in the Windows Update Catalog and should be discoverable by Windows Update without user interaction. If we're being cautious, however, we could link to the specific [catalogue](https://www.catalog.update.microsoft.com/Search.aspx?q=Future%20Technology%20Devices%20International) drivers that have been signed by FTDI itself ("Future Technology Devices International Ltd").

File: src/morello-console/README.md [hunk 1]
```diff
@@ -46,3 +46,14 @@ Typical command lines to access the MCC and Morello consoles are:
...
+
+## Connecting from Windows
+
+Windows requires the installation of FTDI drivers to access the Morello board's USB ports, and the OS should identify the appropriate driver version once the board is connected. After installing the drivers and powering the board on for the first time, access to the MCC console can be tested via the following:
+```powershell
+$port = New-Object System.IO.Ports.SerialPort COM3,115200,None,8,one
+$port.Open()
+$port.WriteLine("?")
+$port.ReadExisting()
+$port.Close()
```
Comment (jrtc27) [inline src/morello-console/README.md:58]: This doesn't really tell you how to actually use the console interactively. I don't think we want to be documenting this way. Documenting a more interactive way to do it on Windows seems fine though, e.g. with PuTTY.

Comment (elisehaldane) [reply src/morello-console/README.md:58]: Built-in methods (PowerShell/CMD) generally seem prudent to include for any OS, but they're far from ideal for this on Windows. Would you suggest omitting PowerShell altogether or keeping it for posterity? Documenting a PuTTY example does seem like good advice, either way.

**Comment by kwitaszczyk** [comment]
I think we can merge it now and improve independently if required.

**Pull request merged**